### PR TITLE
check_grep.sh part ??: adds all global vars to the global controller

### PR DESCRIPTION
**Pull request by harryob**
per title really

god willing no player facing changes

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/shuttles/marine_ferry.dm [hunk 1]
```diff
@@ -74,10 +74,10 @@
 		return //Kill it so as not to repeat
 
 /datum/shuttle/ferry/marine/proc/load_datums()
-	if(!(info_tag in s_info))
+	if(!(info_tag in GLOB.s_info))
```
Comment (fira) [inline code/modules/shuttles/marine_ferry.dm:77]: doesnt seem like anything puts anything in this, could probbaly be cut and /proc/loadShuttleInfoDatums too

File: html/changelogs/archive/2020-10.yml [hunk 2]
```diff
@@ -147,7 +147,7 @@
   - rscadd: remixed smartgun firing audio.
 2020-10-14:
   Vampmare:
-  - tweak: Reworked the RoleAuthority and rolling of roles. Special roles are now
+  - tweak: Reworked the GLOB.RoleAuthority and rolling of roles. Special roles are now
```
Comment (fira) [inline html/changelogs/archive/2020-10.yml:150]: CL edit

File: html/changelogs/archive/2020-10.yml [hunk 3]
```diff
@@ -380,7 +380,7 @@
       Trap cooldown per hostile living mob hit.'
 2020-10-25:
   Jeser:
-  - bugfix: Readded Debug RoleAuthority verb that was unintentionally deleted some
+  - bugfix: Readded Debug GLOB.RoleAuthority verb that was unintentionally deleted some
```
Comment (fira) [inline html/changelogs/archive/2020-10.yml:383]: CL edit

**Review by fira — CHANGES_REQUESTED** [review]
Finally, just found a few minor things to cut pretty much

File: code/__DEFINES/atmospherics.dm [hunk 4]
```diff
@@ -38,7 +38,7 @@
 #define GAS_TYPE_CO2 "carbon dioxyde"
 
 /// This was a define, but I changed it to a variable so it can be changed in-game.(kept the all-caps definition because... code...) -Errorage
-var/MAX_EXPLOSION_RANGE = 14
+GLOBAL_VAR_INIT(MAX_EXPLOSION_RANGE, 14)
```
[the remark this answers is not in the document]
Comment (fira) [reply code/__DEFINES/atmospherics.dm:41]:
```suggestion
```

File: code/__HELPERS/cmp.dm [hunk 5]
```diff
@@ -53,7 +53,7 @@ var/cmp_field = "name"
 	if (!.)
 		. = B.qdels - A.qdels
 
-var/atom/cmp_dist_origin=null
+GLOBAL_DATUM(cmp_dist_origin, /atom)
```
[the remark this answers is not in the document]
Comment (fira) [reply code/__HELPERS/cmp.dm:56]:
```suggestion
```

File: code/_globalvars/global_lists.dm [hunk 6]
```diff
@@ -132,13 +129,13 @@ GLOBAL_LIST_INIT(surgical_patient_types, setup_surgical_patient_types())
...
 
-var/global/list/active_areas = list()
-var/global/list/all_areas = list()
+GLOBAL_LIST_EMPTY(active_areas)
+GLOBAL_LIST_EMPTY(all_areas)
 
-var/global/list/turfs = list()
-var/global/list/z1turfs = list()
+GLOBAL_LIST_EMPTY(turfs)
+GLOBAL_LIST_EMPTY(z1turfs)
```
[the remark this answers is not in the document]
Comment (fira) [reply code/_globalvars/global_lists.dm:136]:
```suggestion
```

File: code/modules/shuttles/marine_ferry.dm (hunk 1 above)
Comment (fira) [reply code/modules/shuttles/marine_ferry.dm:77]: duplicate

**Review by fira — APPROVED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by fira** [comment]
Let's TM this a few rounds quick to make sure no accident and get it in

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Pull request merged**